app/models/cinema_hall.rb

```diff
@@ -2,10 +2,30 @@
 
 class CinemaHall < ApplicationRecord
   has_many :screenings, dependent: :destroy
+  has_many :seats, dependent: :destroy
   before_save :assign_cinema_hall_column_and_rows
+  after_save :assign_seat_names
```

Comment:
Using hooks is generally not a good idea. I know that currently, we don't have better mechanisms to handle this use case, but consider [this article](https://samuelmullen.com/2013/05/the-problem-with-rails-callbacks/) as an introduction to better architectural decision here.

Reply:
Are you still using this hook?